ImageNet/train/fastai_pytorch.tsv

```diff
@@ -1,46 +1,46 @@
 epoch	hours	top1Accuracy	top5Accuracy
-0	0.03601963861111111	6.335	19.138
-1	0.06880011055555556	17.944	40.332
-2	0.09799499888888889	23.665	49.005
-3	0.12732031583333334	29.668	55.522
-4	0.15941369916666667	33.366	59.668
-5	0.19305480416666668	30.713	56.451
-6	0.2219483125	37.529	64.511
-7	0.25109444138888887	42.869	69.138
-8	0.2805305605555556	44.959	71.559
-9	0.31185991444444444	40.365	67.612
-10	0.3407848383333333	42.803	69.420
-11	0.3699388372222222	42.454	68.773
-12	0.4002340933333334	45.456	72.272
-13	0.42938565138888884	45.406	73.499
-14	0.4585297086111111	49.254	74.710
-15	0.48779424694444445	48.093	73.831
-16	0.5287843358333333	45.572	71.675
-17	0.56932719	48.010	74.411
-18	0.6687784997222221	46.318	72.869
-19	0.7661416472222222	52.371	78.988
-20	0.8612041186111111	56.020	81.509
-21	0.9557247583333333	70.265	89.917
-22	1.0512144397222223	71.692	90.133
-23	1.145943983888889	70.083	89.834
-24	1.2483195533333333	70.730	90.282
-25	1.342536527222222	71.957	91.725
-26	1.4367986494444445	71.542	89.768
-27	1.5312345966666667	72.206	90.514
-28	1.625720830277778	71.410	90.514
-29	1.7201986619444445	71.393	90.431
-30	1.8224384877777777	70.962	90.962
-31	1.915672317777778	71.708	90.746
-32	2.019930609722222	70.846	90.265
-33	2.1139093641666666	71.658	90.597
-34	2.208938075555556	71.061	90.216
-35	2.3096664283333332	74.494	92.653
-36	2.4041665241666665	74.163	92.007
-37	2.5022614369444445	74.511	92.206
-38	2.6044323761111112	74.411	92.040
-39	2.698724052777778	74.925	92.421
-40	2.794010547777778	75.357	92.388
-41	2.9633895408333335	75.768	93.003
-42	3.1232502747222224	75.884	92.972
-43	3.2913005908333335	75.919	93.045
-44	3.4616945625	76.042	93.053
+0	0.03617896611111111	7.040	21.408
+1	0.06521215444444445	13.872	31.856
+2	0.09530189416666666	25.008	50.560
+3	0.14340053249999998	22.704	45.200
+4	0.17227937055555556	32.832	59.280
+5	0.2013068286111111	32.592	59.008
+6	0.23029296222222223	40.080	67.408
+7	0.2658237766666667	38.224	65.184
+8	0.29882442277777777	41.344	68.432
+9	0.3276660186111111	40.704	67.072
+10	0.3562790472222222	44.096	70.336
+11	0.39372730833333336	43.904	69.584
+12	0.42229111444444445	38.816	65.712
+13	0.45084065944444446	39.520	66.960
+14	0.4794358952777778	42.640	70.208
+15	0.5098701902777778	42.448	68.416
+16	0.5416913427777778	40.160	67.760
+17	0.5703782808333334	44.192	70.448
+18	0.6707612511111111	51.856	77.888
+19	0.7648842469444445	49.728	75.264
+20	0.858128875	42.592	69.136
```

Comment:
Hi @jph00, just curious, do you know why the first 20 epochs (150+ seconds per epoch) are twice as fast as the next 20 (340+ seconds per epoch)?

Is it something pytorch specific or would it be something you are doing different apart from learning rate changes?